docs/conf.py

```diff
@@ -257,7 +257,9 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
 html_css_files = ["custom.css"]
-html_js_files = ["pydata-icon.js", "custom-icon.js"]
+html_js_files = [
+    ("custom-icon.js", {"defer": "defer"}),
```

Comment:
See note in PR description. Since this PR changes fontawesome.js to defer, it must also change custom-icon.js (or any code that depends on fontawesome.js to defer)